includes/request.js

```diff
@@ -34,5 +46,11 @@ module.exports = {
       headers: {'User-Agent': userAgent},
       json: true,
       gzip: true});
+  },
+  isOk: function(resp, callback) {
```

Comment:
ok so these names are a bit confusing, I was under the impression they were booleans.
My suggestions are terribly wordy but I think they articulate the purpose, perhaps there is a suitable compromise :)
`checkResponseAndCallbackError`
`checkResponseAndCallbackFatalError`